### PR TITLE
(fix) [SD-2725] Locator code works but it would be nice to separate the dropdown into two lists - Australia and Rest of World

### PR DESCRIPTION
**Pull request by sivakuna-aap**
##### Instructions to the Merger(s):
1. Underlying structure of Vocabulary is changed. After merging please drop vocabularies collection and recreate collection using `vocabularies.json` file.

### CODE REVIEW

**Comment by sivakuna-aap** [comment]
@superdesk/developers

File: client/app/scripts/superdesk-authoring/metadata/metadata.js [hunk 1]
```diff
@@ -193,6 +193,14 @@ function MetadataDropdownDirective() {
                 _.extend(scope.item, o);
                 scope.change({item: scope.item});
             };
+
+            $timeout(function() {
```
Comment (akintolga) [inline client/app/scripts/superdesk-authoring/metadata/metadata.js]: Is the $timeout necessary here?

Comment (sivakuna-aap) [inline client/app/scripts/superdesk-authoring/metadata/metadata.js]: Yes, what I observed is, sometimes the variables declared in directive scope doesn't get values until the digest completes.

File: client/app/scripts/superdesk-authoring/metadata/metadata.js [hunk 2]
```diff
@@ -193,6 +193,14 @@ function MetadataDropdownDirective() {
...
+                if (scope.list && scope.field === 'place') {
+                    scope.places = _.groupBy(scope.list, function(place) {
+                        return place.country === 'Australia' ? 'Australia' : 'Rest of World';
```
Comment (amagdas) [inline client/app/scripts/superdesk-authoring/metadata/metadata.js]: `gettext`?

**Comment by akintolga** [comment]
:+1:

File: client/app/scripts/superdesk-authoring/metadata/metadata.js (hunk 2 above)
Comment (ioanpocol) [inline client/app/scripts/superdesk-authoring/metadata/metadata.js]: Can you make the 'Australia' string as a setting value in settings file?

**Comment by ioanpocol** [comment]
:+1:  except comments

File: client/app/scripts/superdesk-authoring/metadata/metadata.js (hunk 2 above)
Comment (sivakuna-aap) [inline client/app/scripts/superdesk-authoring/metadata/metadata.js]: We had a discussion about this. I will move it to vocabularies itself. Adding it there will give more flexibility in terms if tomorrow we want to group them into more than 2.

Comment (ioanpocol) [inline client/app/scripts/superdesk-authoring/metadata/metadata.js]: Yes, this is better

**Comment by sivakuna-aap** [comment]
@ioanpocol replaced hard coding with vocabulary configuration.

**Comment by ioanpocol** [comment]
:+1:

**Pull request merged**